### PR TITLE
New azure doc- Create Azure Kubernetes cluster

### PR DESCRIPTION
**Pull request by BaileyLawson**

### CODE REVIEW

File: articles/azure/azs-how-create-aks.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+   ![Create Kubernetes Cluster > Basics](images/azs-browser-create-aks-basics.png)
+
+6. In the *Kubernetes Cluster Settings* step, enter the following information and click **OK**:
+
+   **Linux VM Admin Username** - The username for the Linux Virtual Machines that are part of the Kubernetes Cluster.
+
+   **SSH public key** - SSH public key used for auth to all Linux machines created as part of the the Kubernetes cluster.
+
+   **Master Profile DNS Prefix** - This must be a region-unique name e.g. k8s-12345. Try to choose the same name as the resource group as best practice.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: for example (not e.g.)

File: articles/azure/azs-how-create-aks.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+   **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   **Location** - This will be "frn00006", which is the location of the Azure Stack.
+
+   ![Create Kubernetes Cluster > Basics](images/azs-browser-create-aks-basics.png)
+
+6. In the *Kubernetes Cluster Settings* step, enter the following information and click **OK**:
+
+   **Linux VM Admin Username** - The username for the Linux Virtual Machines that are part of the Kubernetes Cluster.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: virtual machines (no uppercase letters)

File: articles/azure/azs-how-create-aks.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    ![List of templates in Compute blade](images/azsp_computeblade.png)
+
+5. In the *Create Kubernetes Service cluster* blade, in the *Basics* step, enter the following information and click **OK**:
+
+   **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   **Resource group** - Select an existing resource group, or create a new one by typing a name for your new resource group.
+
+   **Location** - This will be "frn00006", which is the location of the Azure Stack.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: use backticks instead of quotes

File: articles/azure/azs-how-create-aks.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+   ![Create Kubernetes Cluster > Basics](images/azs-browser-create-aks-basics.png)
+
+6. In the *Kubernetes Cluster Settings* step, enter the following information and click **OK**:
+
+   **Linux VM Admin Username** - The username for the Linux Virtual Machines that are part of the Kubernetes Cluster.
+
+   **SSH public key** - SSH public key used for auth to all Linux machines created as part of the the Kubernetes cluster.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: authentication (not auth)

File: articles/azure/azs-how-create-aks.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+   **Master Profile DNS Prefix** - This must be a region-unique name e.g. k8s-12345. Try to choose the same name as the resource group as best practice.
+
+   **Kubernetes Master Pool Profile Count** - The number of master nodes for the Kubernetes cluster. This value should be an odd number.
+
+   **The VMSize of Kubernetes node VMs** - The VM size of each master node. For information about the different available VM sizes, see <https://docs.microsoft.com/en-gb/azure/azure-stack/user/azure-stack-vm-sizes>
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: Should there be a space between VM and Size?

File: articles/azure/azs-how-create-aks.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+   **Kubernetes Node Pool Profile Count** - The number of agents for the Kubernetes cluster.
+
+   **The VMSize of Kubernetes node VMs** - The VM size of each agent node.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: VMSize? Is that how it looks in the interface? If so, I guess leave it as it is

File: articles/azure/azs-how-create-aks.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+7. In the *Summary* step, check that the information is correct, then click **OK**.
+
+8. Click **Create**.
+
+9. You can monitor the progress of your cluster's deployment by clicking the **Notifications** icon.
+
+    ![Notification showing cluster deployment in progress](images/azsp_createvm_progress.png)
+
+10. After your cluster has been deployed, you can view it by clicking **Resource Groups** in the favourites panel, then selecting the resource group which contains the cluster.
```
Comment (suehighmoor) [inline articles/azure/azs-how-create-aks.md]: use that instead of which

File: articles/azure/azs-how-create-aks.md (hunk 5 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-create-aks.md]: That's the name of it in the portal so I think it should be left as it is

File: articles/azure/azs-how-create-aks.md (hunk 6 above)
Comment (BaileyLawson) [reply articles/azure/azs-how-create-aks.md]: This is how it appears on the portal, as bad as that is...

**Pull request merged**